### DISCUSSION TITLE
Change netty's thread

### DISCUSSION THREAD

**Discussion by andreasdc** [netty/netty#12163] [Q&A]
Hey, is it possible to set netty's thread to 1 of my java.lang.Thread? Incoming packets are the most important.

**Comment by johnou** [comment]
Use your own thread factory?

**Reply by andreasdc** [reply]
> Use your own thread factory?

I just want to have 1 thread.

**Reply by johnou** [reply]
if you are using nio you can configure the amount of threads with this constructor https://netty.io/4.0/api/io/netty/channel/nio/NioEventLoopGroup.html#NioEventLoopGroup-int-

**Reply by andreasdc** [reply]
> if you are using nio you can configure the amount of threads with this constructor https://netty.io/4.0/api/io/netty/channel/nio/NioEventLoopGroup.html#NioEventLoopGroup-int-

Yes, and I want to maintain other actions in this thread.

**Reply by johnou** [reply]
Should be able to do that in combination with your own thread factory then, word of warning though.. make sure those other operations are non blocking or your event loop will stop processing events resulting in timeouts etc.

**Reply by andreasdc** [reply]
I got this error:
io.netty.channel.AbstractChannel - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x18f6452b]
